(2) Vox Populi/Database Changes/Art/UnitArtChanges.sql

```diff
@@ -34,6 +35,7 @@ VALUES
 	('ART_DEF_UNIT_HEAVY_SKIRMISH', 'ART_DEF_UNIT_MEMBER_HEAVY_SKIRMISH', 5),
 	('ART_DEF_UNIT_CUIRASSIER', 'ART_DEF_UNIT_MEMBER_CUIRASSIER', 5),
 	('ART_DEF_UNIT_ARMORED_CAR', 'ART_DEF_UNIT_MEMBER_ARMORED_CAR', 2),
+	('ART_DEF_UNIT_U_POLYNESIAN_MAORI_WARRIOR', 'ART_DEF_UNIT_MEMBER_U_POLYNESIAN_MAORI_WARRIOR', 6),
```

Comment:
Is it `ART_DEF_UNIT_MEMBER_U_POLYNESIAN_MAORI_WARRIOR` or `ART_DEF_UNIT_U_POLYNESIAN_MAORI_WARRIOR`?